<?xml version="1.0"?>

<launch>
    <node pkg="siesta" type="quick_read_node" name="quick_read_node" output="screen"/>
    <!--node pkg="siesta" type="set_point_publisher_node" name="set_point_publisher_node" output="screen"/-->
</launch>